gtsfm/utils/geometry_comparisons.py

```diff
@@ -263,17 +263,41 @@ def compare_global_poses(
     return rotations_equal and translations_equal
 
 
+def compute_rotation_angle_measurement_consistency(
+    i2Ri1: Optional[Unit3], wRi2: Optional[Pose3], wRi1: Optional[Pose3]
+) -> Optional[float]:
+    """Compute angle between a [ TODO ] between 2 poses.
+
+    Given a relative rotation measurement from i2 to i1, and the estimated global rotations of
+    i1 and i2, returns the angular difference between the relative vs. synthetic measurements.
+
+    Args:
+        i2Ri1: Relative rotation measurement.
+        wRi2: Global rotation of camera i2.
+        wRi1: Global rotation of camera i1.
+
+    Returns:
+        Angle between two-view measurement and synthetic relative rotation in degrees.
+    """
+    if i2Ri1 is None or wRi2 is None or wRi1 is None:
+        return None
+
+    # Synthetic measurement.
+    i2Ri1_synthetic = wRi2.between(wRi1)
```

Comment:
I dont think we need to call this "synthetic measurement", as it comes from the estimate. It is basically the consistency between the estimate and the measurement.

Reply:
I think it gets a little confusing though, since the measurement is also an estimate (it's an estimated measurement).

So I think the dichotomy has to be what kind of 2-view measurement is it?
(1) estimated by system directly
(2) synthetically derived by other system outputs

Reply:
@akshay-krishnan I updated the name to `i2Ri1_derived`, hopefully this is clearer now